Updated deprecated Svelte tooling information (#26346)
mdn/content: files/en-us/learn/tools_and_testing/client-side_javascript_frameworks/svelte_getting_started/index.md

- Highly interactive pages or complex visualizations: If you are building data-visualizations that need to display a large number of DOM elements, the performance gains that come from a framework with no runtime overhead will ensure that user interactions are snappy and responsive.
- Onboarding people with basic web development knowledge: Svelte has a shallow learning curve. Web developers with basic HTML, CSS, and JavaScript knowledge can easily grasp Svelte specifics in a short time and start building web applications.

> The Svelte team has also launched [SvelteKit](https://kit.svelte.dev), a web framework that extends Svelte, enabling developers to create robust and performant web applications using Svelte. It contains basic features found in modern web frameworks, such as filesystem-based routing and server-side rendering (SSR). The more advanced features include page-specific rendering modes, offline support, and more. For more information about SvelteKit, head for the [interactive tutorial](https://learn.svelte.dev) and/or read the [documentation](https://kit.svelte.dev/docs).

Reviewer: Minor rewording here:

```suggestion
The Svelte team launched [SvelteKit](https://kit.svelte.dev), a framework for building web applications using Svelte. It contains features found in modern web frameworks, such as filesystem-based routing, server-side rendering (SSR), page-specific rendering modes, offline support, and more. For more information about SvelteKit, see the [official tutorial](https://learn.svelte.dev) and [documentation](https://kit.svelte.dev/docs).
```
